src/services/infrastructure/url-generator/url.generator.service.ts

```
@@ -294,6 +295,12 @@ export class UrlGeneratorService {
       case ProfileType.USER_GROUP:
         // to do: implement and decide what to do with user groups
         return `${this.endpoint_cluster}`;
+      case ProfileType.KNOWLEDGE_BASE:
+        const vc =
+          await this.getVirtualContributorFromKnowledgeBaseProfileOrFail(
+            profile.id
+          );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)